front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -97,12 +97,17 @@ export default function AssistantBuilder({
   const [submitEnabled, setSubmitEnabled] = useState(false);
   const [disableUnsavedChangesPrompt, setDisableUnsavedChangesPrompt] =
     useState(false);
+
+  // The 4 kind of errors that can be displayed in the assistant builder
   const [assistantHandleError, setAssistantHandleError] = useState<
     string | null
   >(null);
   const [instructionsError, setInstructionsError] = useState<string | null>(
     null
   );
+  const [descriptionError, setDescriptionError] = useState<string | null>(null);
+  const [actionsErrors, setActionsErrors] = useState<string[]>([]);
```

Comment:
Why putting a list on the state when we only use it as a boolean to know if there's an error?